scripts/extensions/ai-widget/src/requests.ts

```diff
@@ -0,0 +1,12 @@
+import {IArticle, ISuperdesk} from 'superdesk-api';
+
+export interface IRequests {
+    generateHeadlines?: (article: IArticle, superdesk: ISuperdesk) => Promise<Array<string>>;
+    generateSummary?: (article: IArticle, superdesk: ISuperdesk) => Promise<string>;
+}
+
+export const requests: IRequests = {};
+
+export function setRequests(_requests: IRequests) {
```

Comment:
`configure` or `setup` would be a better name. Then `IRequests` would be `IConfigurationOptions`